docs/design/specs/runtime-async.md

```diff
@@ -43,42 +45,24 @@ Other restrictions are likely to be permanent, including
-
-The second async signature is implicit and runtime-generated and is hereafter referred to as the "Task-equivalent" signature. It is generated based on the primary signature. The transformation is as follows:
-* If the async return type is void, the return type of the Task-equivalent signature is the type of the async custom modifier.
-* Otherwise, the Task-equivalent return type is the custom modifier type (either ``Task`1`` or ``ValueTask`1``), substituted with the async return type.
+Async methods are capable of calling certain methods using an "async signature." The target method must return one of `Task`, `ValueTask`, `Task<T>`, or `ValueTask<T>`.
 
-It is an error to explicitly declare a method with the same signature as an async method's synthesized "Task-equivalent" signature.
+The async signature is generated based on the primary signature. The transformation is as follows:
+* If the target method return type is `Task` or `ValueTask`, the return type of the secondary signature is `void` with the first custom modifier on the return type being the original type.
+* Otherwise, the return type is the type argument of the return type (either ``Task`1`` or ``ValueTask`1``) with the first custom modifier on the return type being the original type.
 
-Unlike async methods, sync MethodDefs do not always generate two member definitions. However, if the sync MethodDef signature would match the "Task-equivalent" signature of an async method, an async definition is also synthesized. More precisely, sync methods that have a return type of `System.Threading.Task`, `System.Threading.ValueTask`, `System.Threading.Task<T>`, or `System.Threading.ValueTask<T>` and have parameters meeting any further requirements of an async method definition, will generate an equivalent async definition corresponding to the inversion of the rules for generating the async method "Task-equivalent" signature.
+It is an error to declare a method with an "async signature".
 
 _[Note: these rules operate before generic substitution, meaning that a method which only meets requirements after substitution would not be considered as valid.]_
 
-### I.8.10.2 Method inheritance
-
-For the purposes of inheritance and hiding, both async signatures ([### I.8.6.1.5 Method signatures]) are used for hiding and overriding, but cannot be configured separately.
-
-### II.10.3.2 The .override directive
-
-Async methods participate in overrides through both definitions (both signatures). An async method with a .override overrides the target method signature, as well as the secondary "Task-equivalent" signature if applicable. An async method may also override a sync method matching the "Task-equivalent" signature, if an async signature is not present on the base class. A sync method may also override an async method's "Task-equivalent" signature. This will behave as though both the async and "Task-equivalent" methods have been overridden.
-
```

Comment:
I like that a bunch of text got removed. Fewer rules and cases that need to be described a good sign for a spec.